custom_components/rpi_gpio/const.py

```diff
@@ -0,0 +1,4 @@
+
+DOMAIN = "gpiod"
```

Comment:
```suggestion
DOMAIN = "rpi_gpiod"
```
I know this is now supporting other platforms, but the name needs to match the intergration

Reply:
fixed.